README.md

```diff
@@ -5,5 +5,99 @@ Mage-tools is an opinionated set of [mage](https://github.com/magefile/mage) tar
 [![Release](https://github.com/einride/mage-tools/actions/workflows/release.yml/badge.svg)](https://github.com/einride/mage-tools/actions/workflows/release.yml)
 
 ## Requirements
--	[Go](https://golang.org/doc/install) >= 1.17
--	[GNU Make](https://www.gnu.org/software/make/)
+
+- [Go](https://golang.org/doc/install) >= 1.17
+- [GNU Make](https://www.gnu.org/software/make/)
+
+## Getting started
+
+To initilize mage-tools in a repository, just run:
+
+```bash
+go run -mod=mod go.einride.tech/mage-tools -init
+```
+
+Run `make`
+
+## Usage
+
+All mage imports, and targets within the magefiles, gets written to a default Makefile which is always imported. It can also generate specific makefiles to only be imported in certain places like in a Makefile that should only have terraform targets.
```

Comment:
Hmm, are we running with the latest @einride/prettier-config? It should line wrap automatically too.

Reply:
I just did that, at least the version we have defined in the target

Reply:
Something might be broken in the upstream config! This change should enable prose wrapping: https://github.com/einride/prettier-config/commit/ccf90f0a8d7fb073955daff7e8d54c2c01fd848d

@filiptammergard Not urgent, but something for us to look into.

Reply:
We're only installing @einride/prettier-config in the target, but we're not actually using it. To use it we need to create a `.prettierrc.js` file and register it there. Something like this: https://github.com/einride/mage-tools/pull/43 I didn't get it to work in my PR, probably because I'm not familiar with this way of writing tools, but it's a step in the right direction I think. Perhaps you can see what I missed? @odsod @viktorvoltaire

Reply:
@filiptammergard Nice one - I read that PR and I think I know how to get it working - see comment